mirror-repos-branch

```diff
@@ -41,6 +41,12 @@ trap '{ export EXT="$?"; rm -rf "${REPOS_DIR}" && exit "${EXT}"; }' EXIT
 
 cd "${REPOS_DIR}"
 
+# Check if we are in the monorepo case
+[ ! -d scripts ] && git clone --recurse-submodules "git@github.com:flatcar/scripts"
```

Comment:
We have `set -euo pipefail`. If `scripts` directory exists, then `[[ ! -d scripts ]]` will evaluate to false and the script will quit.

Reply:
The `||` and `&&` operators suppress `-e` even though they still propagate the exit code.

Reply:
Oh indeed. Nevermind then.